src/constants.ts

```diff
@@ -1,23 +1,30 @@
-const BIEL_LINK = `https://bibleineverylanguage.org/`;
+const BIEL_LINK = "https://bibleineverylanguage.org/";
 const DESKTOP_BREAKPOINT = 900;
 const DEFAULT_ULB_LINE_HEIGHT = 200;
 const DEFAULT_ULB_FONT_SIZE = 20;
 const DEFAULT_INFORMATIONPANEL_LINE_HEIGHT = 150;
 const DEFAULT_INFORMATIONPANEL_FONT_SIZE = 20;
-const MORPHOLOGY_FILE_PATH = "https://content.bibletranslationtools.org/WycliffeAssociates/en_gwt/raw/branch/master/02_morphology_files";
-const FEEDBACK_HANDLER_URL = "https://wagwtfeedbackhandler.azurewebsites.net/api/MissingHandler";
-const FEEDBACK_HANDLER_KEY = "LFRaycifocGWKg-97QV13FFYNxMpXop847cwTlxdgXb2AzFuQXNxbg==";
+const MORPHOLOGY_FILE_PATH =
+	"https://content.bibletranslationtools.org/WycliffeAssociates/en_gwt/raw/branch/master/02_morphology_files";
+const FEEDBACK_HANDLER_URL =
+	"https://wagwtfeedbackhandler.azurewebsites.net/api/MissingHandler";
```

Comment:
I personally find it preferable to consolidate urls across the network as environment variables.

Reply:
Agreed.